src/dbnode/integration/options.go

```diff
@@ -81,6 +81,9 @@ const (
 
 	// defaultWriteNewSeriesAsync inserts, and index series' synchronously by default.
 	defaultWriteNewSeriesAsync = false
+
+	// defaultReportInterval is the default time interval of reporting metrics within the system.
+	defaultReportInterval = time.Second
```

Comment:
nit: can we have the default be 0, then only use this value if it has been overridden in the tests that should explicitly reduce it? Just a little paranoid that changing this from 15sec -> 1sec by default might make some other integration tests less useful (e.g. if they were testing interplay on longer reports or something along those lines)